### PR TITLE
feat: adding support for beta language flags in cli

### PR DESCRIPTION
**Pull request by shirlupo**
- [ ] Ready for review
- [ ] Linked to Jira ticket

### CODE REVIEW

**Comment by pkey** [comment]
same comment as in previous PR:
- more context is always helpful for reviewer
- we also tend to include the ticket number in commit or at least PR name - helps to navigate from JIRA to the PR.

File: src/bundles.ts [hunk 1]
```diff
@@ -183,15 +183,32 @@ interface CreateBundleFromFoldersOptions extends ConnectionOptions, AnalyzeFolde
...
   const resp = await getFilters(baseURL, source, undefined, requestId);
   if (resp.type === 'error') {
     throw resp.error;
   }
-  const supportedFiles = resp.value;
-  emitter.supportedFilesLoaded(supportedFiles);
-  return supportedFiles;
+  const supportedFilesFromApi = resp.value;
+  //Given supported languages from 'registy'
+  if (languages) {
```
Comment (pkey) [inline src/bundles.ts:199]: question: does this basically check the response (of supported languages) and verifies whether CLI options are not 💩 ?

File: src/bundles.ts [hunk 2]
```diff
@@ -183,15 +183,32 @@ interface CreateBundleFromFoldersOptions extends ConnectionOptions, AnalyzeFolde
...
+    let supportedFiles: SupportedFiles = {} as any;
+    supportedFiles.configFiles = supportedFilesFromApi.configFiles;
+    supportedFiles.extensions = languages;
+    //For verification only
+    supportedFiles.extensions = supportedFiles.extensions.filter(langSuffix =>
```
Comment (metju90) [inline src/bundles.ts:204]: Yesterday you mentioned renaming `langSuffix` to  `langExtenstion`. Maybe you can do it here too?

File: src/bundles.ts (hunk 1 above)
Comment (metju90) [reply src/bundles.ts:199]: @pkey, What do you mean "by whether CLI options are not 💩 ?" 🤔

Comment (pkey) [reply src/bundles.ts:199]: it looks to me that the code checks languages coming from `options` which are CLI arguments right?

Comment (pkey) [reply src/bundles.ts:199]: Anyways, I approve not to block it.

**Review by pkey — APPROVED** [review]

**Comment by pkey** [comment]
Just make sure to add more context to PRs next time @shirlupo as I mentioned in the previous comment.

File: src/bundles.ts (hunk 1 above)
Comment (shirlupo) [reply src/bundles.ts:199]: @pkey it is also does verification that all supportedLanguages including beta languages are being supported by the backed (/filters)

**Review by metju90 — APPROVED** [review]

**Pull request merged**